deploy/setup.sh

```diff
@@ -3,15 +3,16 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
-ENVCODE=$1
-LOCATION=$2
-PIPELINE_NAME=$3
-ENVTAG=$4
+ENV_CODE=${1:-${ENV_CODE}}
+LOCATION=${2:-${LOCATION}}
+PIPELINE_NAME=${3:-${PIPELINE_NAME}}
+ENV_TAG=${4:-${ENV_TAG}}
+PRE_PROVISIONED_BATCH_ACCOUNT_NAME=${5:-$PRE_PROVISIONED_BATCH_ACCOUNT_NAME}
```

Comment:
Need to update the README.md file for this change as well.

Reply:
Based on previous comment, we can keep this internal and do not have to document this.